Rakefile

```
@@ -4,66 +4,132 @@ namespace :book do
   def exec_or_raise(command)
     puts `#{command}`
     if (! $?.success?)
-      raise "'#{command}' failed"
+      raise "[ERROR] '#{command}' failed"
     end
   end
 
-  desc 'build basic book formats'
-  task :build do
+  def generate_contributors_list(column_size)
+    # Generating preformatted contributors list...
+    `git shortlog -s | grep -v -E "(Straub|Chacon|dependabot)" | cut -f 2- | column -c #{column_size} > book/contributors.txt`
+  end
+
+  def download_locale(locale_file)
+    locale_file_url = "https://raw.githubusercontent.com/asciidoctor/asciidoctor/master/data/locale/#{locale_file}"
+    if not File.exist?(locale_file)
+      puts "Downloading locale attributes file..."
+      l10n_text = URI.open(locale_file_url).read
+      File.open(locale_file, 'w') { |file| file.puts l10n_text }
+      puts " -- Saved at #{locale_file}"
+    else
+      puts "Use existing file with locale attributes #{locale_file}"
+    end
+  end
 
+  # Variables referenced for build
+  lang = 'ru'
+  locale_file = "attributes-#{lang}.adoc"
+  date_string = Time.now.strftime('%Y.%m.%d')
```

```suggestion
  date_string = Time.now.strftime('%d.%m.%Y')
```
Может оставить привычный нам формат даты?

---

👍